packages/app-store/_appRegistry.ts

```diff
@@ -108,10 +113,33 @@ export async function getAppRegistryWithCredentials(userId: number) {
       categories: dbapp.categories,
       credentials: dbapp.credentials,
       installed: true,
+      installCount: mostPopularApps[dbapp.slug] || 0,
       isDefault: usersDefaultApp === dbapp.slug,
       ...(app.dependencies && { dependencyData }),
     });
   }
 
   return apps;
 }
+
+async function getMostPopularApps() {
+  const mostPopularApps = z.array(z.object({ appId: z.string(), installCount: z.number() })).parse(
+    await prisma.$queryRaw`
+    SELECT
+      c."appId",
+      COUNT(*)::integer AS "installCount"
+    FROM
+      "Credential" c
+    WHERE
+      c."appId" IS NOT NULL
+    GROUP BY
+      c."appId"
+    ORDER BY
+      "installCount" DESC
+    `
+  );
```

Comment:
since when do we do that? @zomars

Reply:
Since prisma doesn't have a more performant way to do it AFAIK.

Reply:
is this query fast enough?

Reply:
i also really dont mind to use that space for "Recently added" and sort by created_by. I am not married to "most popular" if the query is too slow

Reply:
It is fast enough